### PR TITLE
[WIP] Fixes #1935: Adds stripe payments initial frontend code

### LINKED ISSUE #1935: Implementing Stripe on Frontend
**I'm submitting a ...**

- [ ] feature request

**Current behaviour:**

**Expected behaviour:**

**Steps to reproduce:**

**Screenshots**

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by abishekvashok**
Fixes #1935

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

Adds initial code to the frontend to setup stripe payments

Signed-off-by: Abishek V AShok <abishekvashok@gmail.com>

### CODE REVIEW

File: frontend/app/templates/components/pricing-component.hbs [hunk 1]
```diff
@@ -6,43 +6,60 @@
...
+                    <div class="shipped icon-box">
+                        <div class="grp-div">
+                            <img class="grp-img" src="/images/blog.png">
+                        </div>
+                        <div class="heading-bravo">More than 200 badges</div>
                         <div class="cb-price">
                             <span class="cb-price-tag">$</span>
-                            <span class="cb-price-number">0.00</span>
-                            <span>for </span>
+                            <span class="cb-price-number">1.00</span>
```
Comment (kushthedude) [inline frontend/app/templates/components/pricing-component.hbs]: The price rights should be given to admin , Create a new field if its not already there

File: frontend/app/templates/components/user-component/user-subscription.hbs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h2 class="ui dividing header">Subscription</h2>
+{{#if session.subscription }}
```
Comment (kushthedude) [inline frontend/app/templates/components/user-component/user-subscription.hbs]: Where are these properties coming from? I don't see any such relation in the model.

File: frontend/app/templates/components/user-subscription.hbs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+{{ else }}
+  <label>You are not subscribed to badgeyay yet. Please subscribe to enjoy premium features and unlimited badges.</label><br><br>
+  <button class="ui basic orange button" {{action "subscribe"}}>Subscribe</button>
+  <button
+    style="background-color:#6772E5;color:#FFF;padding:8px 12px;border:0;border-radius:4px;font-size:1em"
+    id="checkout-button-plan_FMsKtVC2CNPGCz"
+    role="link">
+    Checkout
+  </button>
+  <div id="error-message"></div>
```
Comment (kushthedude) [inline frontend/app/templates/components/user-subscription.hbs]: Whats this empty container for ?

File: frontend/app/templates/components/pricing-component.hbs [hunk 4]
```diff
@@ -6,43 +6,60 @@
         <p class="heading-alpha">Pricing</p>
         <br>
         <div class="inline-columns">
-            <div class="ui stackable two column grid container center aligned">
+            <div class="ui stackable three column grid container center aligned">
```
Comment (kushthedude) [inline frontend/app/templates/components/pricing-component.hbs]: Use class `unstackable`, To have responsiveness in mobile view.

File: frontend/app/templates/components/user-subscription.hbs (hunk 3 above)
Comment (abishekvashok) [reply frontend/app/templates/components/user-subscription.hbs]: Stripe error message container

File: frontend/app/templates/components/user-component/user-subscription.hbs (hunk 2 above)
Comment (abishekvashok) [reply frontend/app/templates/components/user-component/user-subscription.hbs]: The pr is work in progress ;)

File: frontend/app/templates/components/pricing-component.hbs (hunk 1 above)
Comment (abishekvashok) [reply frontend/app/templates/components/pricing-component.hbs]: Price rights are for using badgeyay.

File: frontend/app/templates/components/pricing-component.hbs (hunk 4 above)
Comment (abishekvashok) [reply frontend/app/templates/components/pricing-component.hbs]: Alright

**Comment by abishekvashok** [comment]
Would be done by today

**Pull request closed without merging**